Formula/aws-sso-util.rb

```diff
@@ -0,0 +1,115 @@
+class AwsSsoUtil < Formula
+  include Language::Python::Virtualenv
+
+  desc "Smooth out the rough edges of AWS SSO (temporarily, until AWS makes it better)"
+  homepage "https://github.com/benkehoe/aws-sso-util"
+  url "https://files.pythonhosted.org/packages/5a/ff/28de3b5c4189d244b5bb68c0591875df0a3a47b5b860de2bb9a100f3a60b/aws-sso-util-4.25.0.tar.gz"
+  sha256 "b84fb0b6bb66ee1a5797665cfeceb5d83bf9f3d6a8d8de42687dd65fc06cee8c"
+  license "Apache-2.0"
+  revision 0
```

Comment:
You can remove it, like i suggested, to use the default 0
```suggestion
```

Reply:
Sorry, I didn't notice it the first time. I have removed it now